### ISSUE TITLE
4G network not working anymore since the 31 release

### ISSUE THREAD

**Issue by JPMMPJ** [ubports/ubuntu-touch#1044]
- Device: OnePlus One (Bacon)

- Channel:? (Sorry my first bug report here ; Do not hesitate to let me know what you are waiting for)

- OS : Ubuntu 16.04 (2019-W04)

- Build: 31

### Steps to reproduce

Since the 31 release, 4G network does not work anymore (it worked perfectly until the release 30).

### Expected behavior

4G network working

### Actual behavior

The phone can only work E, H and 3G. Not 4G anymore.

**Comment by JPMMPJ** [comment]
- Channel: Release candidate

**Comment by JPMMPJ** [comment]
Hello Flohack74,

Thank you for your prompt feedback.

I just updated the 32 release. Got the same problem at first. Then checked out my cellular settings and realised it was on "2G/3G (faster)"... I honestly do not recall setting up my phone in that mode... (I mean before doing the 31 release update...)

Anyway, I set it back to "2G/3G/4G (faster)" and everything works just fine now. 4G network is back now. Am really sorry for the confusion...

Thank you very much for your great work Flohack74 and the rest of the Team!!

Best regards,

Jean-Pierre
